### PR TITLE
Ability to put bears on meat spikes.

### PR DESCRIPTION
**Pull request by MMMiracles**
image

Lets players spike a bear in order to get its pelt, as well yield 5 meat instead of the usual 3 from a knife. Less messy, more efficient.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
:+1: 
more uses for meat spikes, more fun

File: html/changelogs/MMMiracles-Bearpelts4everyone [hunk 1]
```diff
@@ -0,0 +1,6 @@
+   author: MMMiracles
```
Comment (Jordie0608) [inline html/changelogs/MMMiracles-Bearpelts4everyone]: You need a .yml extension on this file

**Comment by SconesC** [comment]
@MMMiracles Would you consider making some food recipes unique to bear meat? I'm 90% certain there aren't any

**Comment by MMMiracles** [comment]
@SconesC I can probably look into adding a dish or two, can't exactly guarantee anything great.

**Comment by phil235** [comment]
Kitchen spike code could really use a rewrite. You can destroy the nukedisk by impaling a monkey with the disk in its backpack. But that's for another PR. :+1:

**Comment by Iamgoofball** [comment]
the meat spike is one of those things that could use the new byond icon shit to make those spike sprites

**Comment by RemieRichards** [comment]
@Iamgoofball welcome to yesterday in the sprite request thread where I suggested that.

**Pull request merged**